src/openforms/forms/migrations/0105_data_migrate_form_authentication_backend_options_to_form_authentication_backend.py

```diff
@@ -0,0 +1,97 @@
+# Generated by Django 4.2.20 on 2025-04-23 10:01
+
+from django.db import migrations
+from django.db.migrations.state import StateApps
+
+
+def transform_form_authentication_backends_and_options_to_form_authentication_backend(
+    apps: StateApps, _
+):
+    Form = apps.get_model("forms", "Form")
+    FormAuthenticationBackend = apps.get_model("forms", "FormAuthenticationBackend")
+
+    form_authentication_backends_to_create = []
+    forms_to_update = []
+    forms_with_authentication_backends_or_options = Form.objects.exclude(
+        authentication_backend_options={}, authentication_backends=[]
+    ).iterator()
+
+    for form in forms_with_authentication_backends_or_options:
+        plugins = form.authentication_backends
+        config = form.authentication_backend_options
+
+        # Collect all authentication_backends and authentication_backend_options in one list
+        form_auths_to_create = {}
+        for plugin in plugins:
+            form_auths_to_create[plugin] = {}
+        for plugin, options in config.items():
+            form_auths_to_create[plugin] = options
+
+        for plugin, options in form_auths_to_create.items():
+            # Only create a FormAuthenticationBackend if it doesn't exist already
+            if not FormAuthenticationBackend.objects.filter(
+                form=form, backend=plugin
+            ).exists():
+                form_authentication_backends_to_create.append(
+                    FormAuthenticationBackend(
+                        form=form, backend=plugin, options=options
+                    )
+                )
+
+        form.authentication_backends = []
+        form.authentication_backend_options = {}
```

Comment:
I wouldn't reset these in the data migration - if something goes wrong, there's a snapshot to recover the configuration and the columns will be dropped in a future migration anyway.